etc/kayobe/stackhpc.yml

```
@@ -160,15 +160,23 @@ stackhpc_kolla_ansible_source_version: stackhpc/{{ openstack_release }}
 stackhpc_docker_registry: "{{ pulp_url | regex_replace('^https?://', '') }}"
 
 # Username and password of container registry.
-stackhpc_docker_registry_username: "{{ pulp_username }}"
-stackhpc_docker_registry_password: "{{ pulp_password }}"
+stackhpc_docker_registry_username: "{{ pulp_stack_username if pulp_stack_password is defined and pulp_stack_password | length > 0 else pulp_username }}"
+stackhpc_docker_registry_password: "{{ pulp_stack_password if pulp_stack_password is defined and pulp_stack_password | length > 0 else pulp_password }}"
 
 ###############################################################################
 # Feature flags
 
 # Whether or not to run CIS benchmark hardening playbooks. Default is false.
 #stackhpc_enable_cis_benchmark_hardening_hook:
 
+# Whether or not to check the installed Kayobe version when running Kayobe commands.
+# Default is true.
+stackhpc_enable_kayobe_check: true
```

This needs to be set to false for stackhpc/master
```suggestion
stackhpc_enable_kayobe_check: false
```